### ISSUE TITLE
1.18.2 not working

### ISSUE THREAD

**Issue by litoj** [AMereBagatelle/fabricskyboxes#34]
There seems to have been a breaking change in 1.18.2 and the mod works no longer.

**Comment by iijj22gg** [comment]
Honestly, 1.18.2 broke so many mods I'm going to attempt to fix some of them myself... once I learn how to do it

Here's a NEC stacktrace for this mod:
```Mixin apply for mod fabricskyboxes failed fabricskyboxes.mixins.json:skybox.SkyboxRenderMixin -> net.minecraft.class_761: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException Critical injection failure: @Inject annotation on renderCustomSkyboxes could not find any targets matching 'Lnet/minecraft/class_761;method_3257(Lnet/minecraft/class_4587;Lnet/minecraft/class_1159;FLjava/lang/Runnable;)V' in net.minecraft.class_761. Using refmap fabricskyboxes-1.18-refmap.json [PREINJECT Applicator Phase -> fabricskyboxes.mixins.json:skybox.SkyboxRenderMixin -> Prepare Injections ->  -> handler$zin000$renderCustomSkyboxes(Lnet/minecraft/class_4587;Lnet/minecraft/class_1159;FLjava/lang/Runnable;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V -> Parse]
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Critical injection failure: @Inject annotation on renderCustomSkyboxes could not find any targets matching 'Lnet/minecraft/class_761;method_3257(Lnet/minecraft/class_4587;Lnet/minecraft/class_1159;FLjava/lang/Runnable;)V' in net.minecraft.class_761. Using refmap fabricskyboxes-1.18-refmap.json [PREINJECT Applicator Phase -> fabricskyboxes.mixins.json:skybox.SkyboxRenderMixin -> Prepare Injections ->  -> handler$zin000$renderCustomSkyboxes(Lnet/minecraft/class_4587;Lnet/minecraft/class_1159;FLjava/lang/Runnable;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V -> Parse]
	at Not Enough Crashes deobfuscated stack trace.(1.18.2+build.1)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.validateTargets(InjectionInfo.java:656)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.findTargets(InjectionInfo.java:587)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.readAnnotation(InjectionInfo.java:330)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.<init>(InjectionInfo.java:316)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.<init>(InjectionInfo.java:308)
	at org.spongepowered.asm.mixin.injection.struct.CallbackInjectionInfo.<init>(CallbackInjectionInfo.java:46)
	at jdk.internal.reflect.GeneratedConstructorAccessor18.newInstance(Unknown Source)
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo$InjectorEntry.create(InjectionInfo.java:149)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.parse(InjectionInfo.java:708)
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.prepareInjections(MixinTargetContext.java:1330)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.prepareInjections(MixinApplicatorStandard.java:1043)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:393)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:325)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:247)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	at net.minecraft.client.MinecraftClient.<init>(MinecraftClient:573)
	at net.minecraft.client.main.Main.main(Main:197)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)```

**Comment by AMereBagatelle** [comment]
Resolved by #35
